src/shared/sam/debugger/awsSamDebugConfiguration.ts

```diff
@@ -97,6 +108,7 @@ export function createCodeAwsSamDebugConfig(
 ): AwsSamDebuggerConfiguration {
     const workspaceRelativePath = folder ? getNormalizedRelativePath(folder.uri.fsPath, projectRoot) : projectRoot
     const runtime = runtimeFamily ? getDefaultRuntime(runtimeFamily) : undefined
+    const name = runtime ? `${lambdaHandler}-${runtime}` : `${lambdaHandler}`
```

Comment:
Could you make a common function in this file to maintain consistency across both createXxxDebugConfig methods?